Sources/Commands/TestHelpers.swift

```diff
@@ -0,0 +1,136 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Basics
+import SPMBuildCore
+import TSCBasic
+import Workspace
+
+
+/// Locates XCTestHelper tool inside the libexec directory and bin directory.
+/// Note: It is a fatalError if we are not able to locate the tool.
+///
+/// - Returns: Path to XCTestHelper tool.
+func xctestHelperPath(swiftTool: SwiftTool) throws -> AbsolutePath {
```

Comment:
would it be better encapsulations to have these as static functions on a new "TestHelper" struct rather than global functions?

Reply:
I'll make this change (and call it `TestSupport`).  I kept them as free functions to minimize the change on the call sites, but it's better to keep them inside a type.